dev/tools/merge-pr.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/sh -e
+
+# This script depends (at least) on git and jq.
+# It should be used like this: dev/tools/merge-pr.sh /PR number/
+
+#TODO: check arguments and show usage if relevant
+
+PR=$1
+
+git fetch github
```

Comment:
This script also depends on the remote being called `github`. In my case, it is called `upstream` ^^
It could become a second default argument:

    remote=${2:-$(git remote | grep ^github || git remote | grep ^upstream || echo origin)}

and if it is not provided, it defaults to `github` if it exists, or `upstream` if `github` is not a remote and `upstream` is, or `origin` in the last case.

Reply:
Couldn't it default to the current branch's remote (`git config --get branch.$(git rev-parse --abbrev-ref HEAD).remote`)?

Reply:
Right, did not think of this. That's even better.

Reply:
@SkySkimmer `git config --get branch.master.remote` doesn't seem to work on my config. It returns error code 1 with no output. Any idea why that could be?

Reply:
Never mind, probably because the tracking info was lost after I switched from `trunk` to `master`. Will fix.